python/pyarrow/_parquet.pyx

```diff
@@ -1284,6 +1294,14 @@ cdef shared_ptr[WriterProperties] _create_writer_properties(
             props.encoding(tobytes(column),
                            ParquetEncoding_BYTE_STREAM_SPLIT)
 
+    # col_encoding
+    # encoding map - encode individual columns
+
+    if col_encoding is not None:
+        for column, _encoding in col_encoding.items():
+            props.encoding(tobytes(column),
+                           encoding_enum_from_name(_encoding))
```

Comment:
You can check here if `encoding_enum_from_name(_encoding)` gives `None`, and in that case raising a ValueError

Reply:
Yes, that is meant for number 1) in https://github.com/apache/arrow/pull/11724#discussion_r751195257 right? My question is regarding unknown column name, that is number 2) =)

If for example we have a table with columns `a` and `b` and you specify the encoding for unknown column `c` - it does nothing and I am not sure how to raise an error in this case.

That is if I understood number 2) correctly.

Reply:
Ah, yes, sorry :) That was indeed for unknown encoding, not unknown column. If the underlying C++ code doesn't error for unknown columns, let's maybe leave that for now as is.

Reply:
Ok, done.